configs/settings-example.ini

```diff
@@ -5,7 +5,7 @@
 [general]
 sitename = "Hitchwiki"
 metanamespace = "Hitchwiki"
-debug = false
+debug = true
```

Comment:
are u agreeing on this? or should we create a settings.ini.dev settings.ini.prod ?